modules/dev-tools/src/ts-smoosh/log.js

```diff
@@ -0,0 +1,41 @@
+const Colors = {
```

Comment:
We are using probe.gl logging which has similar features, maybe leave a comment?